ministryofjustice/parliamentary-questions
# CDPT-2200  Remove Early bird question number hyperlinks

## Description

Remove hyperlinks around question UINs displayed on the early bird dashboard.

## Self-review checklist


### Screenshots

**Before**
![eb-hyperlinked](https://github.com/user-attachments/assets/f0ed970e-6823-4305-b35c-77306b54e283)

**After**
![eb-hyperlink-removed](https://github.com/user-attachments/assets/4c9f4a0a-791d-46c8-b477-ed351a32cd9c)

### Related JIRA tickets

https://dsdmoj.atlassian.net/jira/software/c/projects/CDPT/boards/864?assignee=557058%3A2d2f4a33-f427-483f-9c36-338caee60eae&selectedIssue=CDPT-2200

### Deployment


### Manual testing instructions

See the 'early bird' page by going to:
    Settings => Early bird list => Early bird preview
From there you should be able to see the links are gone from each question UIN.  You should also see the same by clicking on the link in the "New PQs to be allocated today" email, if you are a subscriber.
